inc/savedsearch.class.php

```diff
@@ -334,8 +332,8 @@ function cleanDBonPurge() {
    /**
     * Print the saved search form
     *
-    * @param integer $ID      ID of the item
-    * @param array   $options Options:
+    * @param $ID         integer    ID of the item
```

Comment:
This does not follow phpdoc rules; see http://glpi-developer-documentation.readthedocs.io/en/master/codingstandards.html#parameters-documentation